Configuration/TypoScript/Setup/Part4GlobalConfiguration.typoscript

```diff
@@ -3,6 +3,7 @@ config {
   # Adjust the title tag to be displayed as “website - page title”
   pageTitleSeparator = -
   pageTitleSeparator.noTrimWrap = | | |
+  contentObjectExceptionHandler = 0
```

Comment:
I would not always deaktivate this. Maybe test by condition if an admin is logged in first?

Reply:
Like this: https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/ErrorAndExceptionHandling/ProductionExceptionHandler/Index.html#error-handling-oops-an-error-detail-admin